packages/cli/tst/index.test.ts

```diff
@@ -1,12 +1,13 @@
 import { describe, it, expect } from '@jest/globals';
 import child_process from 'node:child_process';
 import path from 'node:path';
+import packageJson from "../package.json"
 
 describe('cli', () => {
     it('should emit the version', () => {
         const binScriptPath = path.resolve(`${__dirname}/../src/index.ts`);
         const results = child_process.execSync(`npx ts-node-esm ${binScriptPath} --version`);
 
-        expect(results.toString()).toEqual('0.0.12\n');
+        expect(results.toString()).toEqual(`${packageJson.version}\n`);
```

Comment:
The unit tests are still asserting that the version is 0.0.12, but the current version is 0.0.16. Would you mind updating the clin unit tests?

Reply:
Here is the console.log of the test: 
{
      binScriptPath: '/Users/aidin/Code/open-source/tybalt/packages/cli/src/index.ts',
      results: <Buffer 30 2e 30 2e 31 32 0a>,
      resultsToString: '0.0.12\n',
      packageJsonVersion: '0.0.16\n',
      isEqual: false
}
as you can see the "results" value is not showing the proper app Version.

Reply:
I think the problem is here: https://github.com/doug-wade/tybalt/blob/master/packages/cli/src/index.ts#L14

I think probably the best thing would be to import packageJson the same way you do here, and then pass the `version` to `.version()`